update_cache.sh

```diff
@@ -36,6 +38,9 @@ for arg in "$@"; do
 			clone_if_not_present $extension.git "https://gerrit.wikimedia.org/r/mediawiki/extensions/$extension"
 			fetch_all $extension.git
 		done
+	elif [ $arg = "services" ]; then
+		clone_if_not_present services/wikidata-query-gui.git "https://github.com/wikimedia/wikidata-query-gui.git"
```

Comment:
I think we should directly clone from `https://gerrit.wikimedia.org/r/wikidata/query/gui` here.

Reply:
Cool, swapped it out now.